app/js/importExport.js

```diff
@@ -0,0 +1,86 @@
+import FileSaver from 'file-saver'
+
+/**
+ * Import the backup of saved tabs and the accounting information.
+ * If any of the required keys in the backup object is missing, the backup will abort without importing the data.
+ * @param {storageLocal} storageLocal is needed to restore the accounting information
+ * @param {tabManager} tabManager is required to initialize it with the imported saved tabs
+ * @param {Event} event contains the path of the backup file
+ */
+const importData = (storageLocal, tabManager, event) => {
```

Comment:
Yes, that is a good point. I was thinking about it but didn't make a final decision. I was torn between alert and toast style notification. What would be your preferred style?